### DISCUSSION TITLE
JavaScript upload issue

### DISCUSSION THREAD

**Discussion by kmorgante** [community/community#29181] [Discussions]
I recently used gh-pages to upload and host my website and my hamburger menu icon isn't working. My mentor said that it's because my JavaScript file wasn't uploaded but I do see my JS file in my gh-pages. Any help with this would be so greatly appreciated! 

Here is my code:
https://github.com/kmorgante/REALWEBSITE

**Comment by airtower-luna** [comment]
You have mixed-up spelling. Your [`index.html` tries to include the script](https://github.com/kmorgante/REALWEBSITE/blob/99e6df7519ecc0434fd200bd39b3dd0bc441976d/index.html#L1562) like this:

```html
  <script src="js/scripts.js"></script>
```

But that file doesn't exist in your repository. What you have is `JS/scripts.js`, notice the upper case directory name. Unlike Windows filesystems URLs are case sensitive. To make it work either change the `src` attribute value to match the directory name, or rename the directory.

**Comment by mohamed653** [comment]
try use relative paths 
instead of ` <img src="assets/icon.png" />`  use   ` <img src="./assets/icon.png" />`
